README.md

```diff
@@ -4,7 +4,7 @@ SPDX-FileCopyrightText: 2017-2019 Free Software Foundation Europe e.V.
 SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
-# reuse
+# REUSE helper tool
```

Comment:
This tool has an identity crisis :p It is reminiscent of the EFF's "certbot" tool, which has finally settled on a name, but has also gone by the names "certbot-auto" and "letsencrypt".

Before accepting this change, I'd rather work out how to properly solve the identity crisis in a separate issue.

Reply:
Good point. However, I think an issue won't be the best place to figure this out. I'd rather like a call with Gabriel etc where we discuss the naming of REUSE and its projects, how the team calls itself and so on. What do you think?

Reply:
Ah, already created #91, but we can discuss this outside of an issue, and simply close the issue when it is solved externally ^^